src/V3EmitCModel.cpp

```diff
@@ -115,6 +115,9 @@ class EmitCModel final : public EmitCFunc {
             if (const AstVar* const varp = VN_CAST(nodep, Var)) {
                 if (varp->isPrimaryIO()) {  //
                     emitVarDecl(varp, /* asRef: */ true);
+                    if(v3Global.opt.emitAccessors()) {
+                        emitVarAccessors(varp);
+                    }
```

Comment:
Does this interleave accessors with the member variables?  I think we should put them in separate sections.

Reply:
Yes it does - did this to avoid duplicating the loop logic. I can copy the loop down though to de-interleave